spec/ioki/apis/endpoints/endpoints_spec.rb

```diff
@@ -24,12 +23,24 @@
 
     it 'raises an error if the argument does not respond_to the interpolation method' do
       expect do
-        described_class.url_elements(
-          'platform',
-          [:id],
-          instance_double(Ioki::Model::Platform::RidePassenger, 'non_id_object')
-        )
-      end.to raise_error(ArgumentError, "#'platform': argument 1 must respond to :id or to be a string")
+        described_class.url_elements([:id], 1)
+      end.to raise_error(NoMethodError, /undefined method `id' for 1:Integer/)
```

Comment:
I think the NoMethodError is as good as the custom error here and it allows to simplify the code.